src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/CaseLookupService.java

```diff
@@ -48,7 +48,9 @@ public CaseData getCaseDataAsAdmin(String caseTypeId, String caseId) throws IOEx
     }
 
     public CaseData getCaseData(String caseTypeId, String caseId, String auth) throws IOException {
+        log.error("Looking up " + caseTypeId + "case with caseId " + caseId + " with auth " + auth);
```

Comment:
Did you mean to leave these in? Maybe not with log.error

Reply:
Good catch, I did not mean to leave these in